### DISCUSSION TITLE
Load external images and other resources during runtime

### DISCUSSION THREAD

**Discussion by lv2lrn4lf** [4ian/GDevelop#3887] [Archive] [answered]
i figure it out it was staring at me the whole time, use baseTexture class resource property to change it:
```
baseTexture.resource
```

**Comment by lv2lrn4lf** [comment, marked as answer]
the answer is above

**Reply by Silver-Streak** [reply]
Hello lv2lrn4lf!

The GDevelop project's github discussions section is more designated for longform discussion by the engine contributors and others around potential feature or codebase enhancements, or other platform bugfixes.

If you have questions on how to use the engine, please keep those to the forum or the discord. Thank you.
